Mathlib/Algebra/Group/WithOne/Defs.lean

```diff
@@ -367,18 +368,40 @@ instance addMonoidWithOne [AddMonoidWithOne α] : AddMonoidWithOne (WithZero α)
           rw [Nat.cast_succ, coe_add, coe_one]
       }
 
+instance instLeftDistribClass [AddZeroClass α] [MulZeroClass α] [LeftDistribClass α]  :
```

Comment:
Does this really need `Zero`?

Reply:
Apparently not, since I changed it to `instance instLeftDistribClass [Mul α] [Add α] [LeftDistribClass α] :` and Lean was fine with it. This also fixed the other problem. I think I got confused between the zero being added by WithZero and the potentially nonexistent zero of the underlying alpha structure.